_data/blog_posts.yml

```diff
@@ -1,3 +1,12 @@
+- title: "A talk on the new AWS Application Load Balancer, updates to ECS, and Kinesis Analytics"
+  url: "https://blog.gruntwork.io/a-talk-on-the-new-aws-application-load-balancer-updates-to-ecs-and-kinesis-analytics-abb599cb3cb8#.vhbek5ce2"
+  thumbnail_url: "https://i.imgur.com/KkniqPD.png"
```

Comment:
If there's a better place to upload this, let me know. I didn't see a way to upload arbitrary images on Medium separate from the post itself.